### PR TITLE
Splash screen

### PR DESCRIPTION
**Pull request by SavBetuel**

### CODE REVIEW

File: app/src/main/java/com/example/hungrywolfs/MainActivity.kt [hunk 1]
```diff
@@ -2,10 +2,15 @@ package com.example.hungrywolfs
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import java.lang.Thread.sleep
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        //sleep(1000)
```
Comment (vasileGrad) [inline app/src/main/java/com/example/hungrywolfs/MainActivity.kt:10]: You can remove this comment

**Review by vasileGrad — DISMISSED** [review]

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -8,14 +8,13 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.HungryWolfs">
+        android:theme="@style/splashScreenTheme">
```
Comment (cristiwd) [inline app/src/main/AndroidManifest.xml:11]: Hmm. 
The app theme should be set on the the application, and the 'splashScreenTheme' should be set only on that specific activity. Also take care of the naming, usually styles start with uppercase.

File: app/src/main/java/com/example/hungrywolfs/MainActivity.kt [hunk 3]
```diff
@@ -6,6 +6,8 @@ import android.os.Bundle
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        setTheme(R.style.Theme_HungryWolfs)
```
Comment (cristiwd) [inline app/src/main/java/com/example/hungrywolfs/MainActivity.kt:10]: Not needed if the theme is set back on the application tag in manifest. Or could be set there on the activity tag.

**Review by cristiwd — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        app:layout_constraintBottom_toBottomOf="parent" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="32dp"
+        android:layout_marginTop="40dp"
+        android:fontFamily="sans-serif-medium"
+        android:text="Hungry\nWolfs"
```
Comment (MGabi) [inline app/src/main/res/layout/activity_splash_screen.xml:26]: Make sure every text is extracted to strings.xml, not hardcoded in XML

**Review by MGabi — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="32dp"
+        android:layout_marginTop="40dp"
+        android:fontFamily="sans-serif-medium"
+        android:text="@string/hungry_wolfs"
+        android:textColor="@color/white"
+        android:textSize="65dp"
```
Comment (cristiwd) [inline app/src/main/res/layout/activity_splash_screen.xml:28]: For text sizes we should use Scale-independent Pixels (**sp**). 
https://developer.android.com/guide/topics/resources/more-resources#Dimension
And also extract it to dimes.xml.

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        android:layout_height="wrap_content"
+        app:srcCompat="@drawable/wolf_image"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="32dp"
```
Comment (cristiwd) [inline app/src/main/res/layout/activity_splash_screen.xml:23]: Please extract these margins to dimes.xml.

**Review by cristiwd — CHANGES_REQUESTED** [review]

**Review by MGabi — APPROVED** [review]

**Review by cristiwd — APPROVED** [review]

**Pull request merged**